resources/qml/Cura.qml

```diff
@@ -301,6 +301,17 @@ UM.MainWindow
                 }
             }
 
+            // A hint for the loaded content view. Overlay items / controls can safely be placed in this area
+            Rectangle {
+                id: mainSafeArea
+                anchors.left: viewOrientationControls.right
+                anchors.right: main.right
+                anchors.top: main.top
+                anchors.bottom: main.bottom
+                visible: false // set to true for debugging only
+                color:"#8000FF00"
+            }
```

Comment:
Debug code can be removed (There is no need for it to be created in "live" code, but it will cost a bit of resources to do so)

Reply:
Replaced by Item